www/caddy/src/opnsense/service/templates/Pischem/Caddy/+TARGETS

```diff
@@ -0,0 +1,3 @@
+Caddyfile:/usr/local/etc/caddy/Caddyfile
+rc.conf.d/caddy:/etc/rc.conf.d/caddy
+syslog-ng.conf.d/caddy.conf:/usr/local/etc/syslog-ng.conf.d/caddy.conf
```

Comment:
the subdir thing looks suspicious and may cause your error with the missing socket

Reply:
I mean I could remove this as template and just include it as static file into the package. I guess that would solve the issue. It's not dynamically changed anyway.

Reply:
I have removed the template and included it as static file. Now the installation works as expected. I tested it.

If theres no reason not to do this I wanne keep it this way, cause it's the most pragmatic fix.

Also additionally, when caddy is removed the file will also vanish due to packet manager cleanup, another benefit.